.github/workflows/pypi-publish.yml

```diff
@@ -36,6 +36,29 @@ jobs:
           python -m pip install setuptools
           python -m pip install build twine
 
+      - name: Normalize locale translations directory names
+        if: ${{ fromJSON(inputs.babel-compile-catalog) }}
+        run: |
+          # Find translations directory dynamically (some packages do not start with 'invenio' e.g., 'flask-security')
+          TRANSLATIONS_DIR=$(find . -mindepth 2 -maxdepth 2 -type d -name "translations" -print -quit)
```

Comment:
Finding the correct translations directory in Python packages isn’t as straightforward as in JS, since the package name doesn’t always match the repo name (e.g. flask-security vs flask-security-invenio).

The current find command works well for typical package structures, but should we consider guarding against directories like `node_modules` or `venv` just in case they contain a stray translations/ folder?